2025-05-14 at 19:40:04 | DEBUG | finance.db.db_utils:get_storage_path:34 | 
	DB_DIR: /Users/vnguye12/Code/projects/finance-api/finance/db
	STORAGE_DIR: /Users/vnguye12/Code/projects/finance-api/finance/db/storage
	file_name: /Users/vnguye12/Code/projects/finance-api/finance/db/storage/finance.db
2025-05-14 at 19:40:04 | DEBUG | finance.db.db_utils:initialize_db:24 | Database already exists at /Users/vnguye12/Code/projects/finance-api/finance/db/storage/finance.db. No action taken.
2025-05-14 at 19:40:13 | INFO | finance.routes.expense:get_expense_by_id:35 | Fetching expense with ID: 16
2025-05-14 at 19:40:13 | DEBUG | finance.db.db_api:__init__:18 | Initializing db_conn and db_name = finance.db
2025-05-14 at 19:40:13 | DEBUG | finance.db.db_utils:get_storage_path:34 | 
	DB_DIR: /Users/vnguye12/Code/projects/finance-api/finance/db
	STORAGE_DIR: /Users/vnguye12/Code/projects/finance-api/finance/db/storage
	file_name: /Users/vnguye12/Code/projects/finance-api/finance/db/storage/finance.db
2025-05-14 at 19:40:13 | DEBUG | finance.db.db_api:get_instance:29 | New DBApi instance CREATED with db_name: finance.db
2025-05-14 at 19:40:13 | DEBUG | finance.db.db_api:get_instance:30 | An instance of DBApi exists
2025-05-14 at 19:40:13 | DEBUG | finance.services.expense_service:get_expense_by_id:26 | expense: None
